snowpack/src/build/build-pipeline.ts

```diff
@@ -144,8 +144,6 @@ async function runPipelineTransformStep(
         } else if (result && typeof result === 'object' && (result as {result: string}).result) {
           output[destExt].code = (result as {result: string}).result;
         }
-        // if source maps disabled, don’t return any
```

Comment:
no sourcemap is assigned to output, so these code is unnecessary.

Reply:
Ah, leave this in. Since this is a transform, we're actually replacing an output entry. If we don't delete the source map, it's no longer valid (better to have no source map, than a broken source map).

In fact, if a source map is set, we need to clear it out since it's no longer valid.

I think an improvement would be:

```diff
if (typeof result === 'string') {
          output[destExt].code = result;
+          output[destExt].map = undefined;
dlete
        } else if (result && typeof result === 'object' && (result as {result: string}).result) {
          output[destExt].code = (result as {result: string}).result;
+          output[destExt].map = undefined;
        }

-  // if source maps disabled, don’t return any
-        if (!sourceMaps) output[destExt].map = undefined;
```

Reply:
good, I modified it.